README.md

```diff
@@ -1,2 +1,11 @@
 # CoC
 Code of Conduct of the Risk of Rain 2 modding community
+
+General server rules
+0: Follow Discord/GitHub ToS for general rules regarding what is and is not acceptable.
+1: Don't be an asshole. This includes, but is not limited to, harrasment,  death threats, generally calling people shitty names, racism, sexism, ect.
+2: What Moderators/Admins/Owners say is final. Period. They get to say what does and doesn't happen on the server. If you believe someone's abusing their power, bring it up with me.
```

Comment:
Remove the last sentence since it cannot be resolved, given there isn't any go to rank they're saying you can voice concerns to.